app/components/candidate_interface/rejection_reasons/rejection_reasons_component.html.erb

```diff
@@ -0,0 +1,33 @@
+<% reasons.selected_reasons.each do |reason| %>
+  <div class="app-rejection">
```

Comment:
It's a shame that we can't reuse https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/components/shared/rejection_reasons/rejection_reasons_component.html.erb as there are only minor design differences, not for this PR but perhaps we should ask design to consolidate the two sets of styles as we're doing a lot of conditional work to deliver two tantalizingly similar designs.